### DISCUSSION TITLE
Improving on the quality of images/videos/audio of old beatmaps

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#13863] [Ideas] [answered]
After downloading most of the oldest maps I've realized just how bad (as to be expected) the quality of the audio and backgrounds are, since it is a low skill  task I would be interested in helping out by spending time finding high qualities recordings and images to "upgrade" these mapsets with so they don't feel so disjointed when compared to newer mapsets. I feel like it would be an appreciated QoL improvement without adding much burden to the dev team since you guys already have so much to do.

**Comment by frenzibyte** [comment]
This is not the right place for such tasks, but not sure if there's any other place you could ask this for.

**Reply by mk56-spn** [reply]
i was unsure where else i could bring it up sorry about that

**Comment by peppy** [comment, marked as answer]
This won't happen. Beatmaps are not updated once uploaded. If this annoys you please avoid older beatmaps.
